merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/AerieAppDriver.java

```diff
@@ -39,14 +42,29 @@ public static void main(final String[] args) {
     final var configuration = loadConfiguration();
     final var stores = loadStores(configuration);
 
+    final var typescriptCodeGenerationService = new TypescriptCodeGenerationService();
+
+    final ConstraintsDSLCompilationService constraintsDSLCompilationService;
+    try {
+      constraintsDSLCompilationService = new ConstraintsDSLCompilationService(typescriptCodeGenerationService);
+    } catch (IOException e) {
+      throw new Error("Failed to start ConstraintsDSLCompilationService", e);
+    }
+
```

Comment:
To help the node process terminate cleanly, the scheduler-server has a shutdown hook that calls the `close` method on the compilation service:

https://github.com/NASA-AMMOS/aerie/blob/b8d8fe322ae688ef89ea63faf2e8c670a0c5af55/scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/SchedulerAppDriver.java#L65

```suggestion
Runtime.getRuntime().addShutdownHook(new Thread(constraintsDSLCompilationService::close)); 
```

Reply:
added